nodejs/eks/nodes/ami.ts

```diff
@@ -210,20 +209,20 @@ export function getOperatingSystem(
 
     const resolvedAmiType = toAmiType(amiType);
     if (!resolvedAmiType) {
-        throw new pulumi.ResourceError(
-            `Cannot determine OS of unknown AMI type: ${amiType}`,
-            parent,
```

Comment:
The new `InputPropertyError` doesn't allow you to specify a `parent`, how is it able to attribute it to the correct resource? Does it not care or is it just somehow smarter than the `ResourceError`?

Reply:
The errors are attached to the resource for which construct/call was invoked. So the parent gets resolved implicitly now.

Reply:
Nice!